practicalai/practical-ai-33.md

```diff
@@ -56,7 +56,7 @@ But yeah, I find it a very good problem for someone who thinks like a physicist,
 
 I published a paper -- I proved some bounds on error propagation in neural nets, and the mathematical modeling I did there was also useful to study biomolecular networks with some friends from the John Hopkins medical school, because it turns out that biomolecular networks are just weighted graphs of nonlinear nodes, just like neural nets...
 
-**Chris Benson:** Now, that's pretty cool. I had never thought of it that way. Okay, so I was gonna ask you that - you had talked about that \[unintelligible 00:14:50.17\] you were dealing with robustness in biological systems, with the technical AI safety... Is that the crossover there, are we getting to that, or am I jumping in?
+**Chris Benson:** Now, that's pretty cool. I had never thought of it that way. Okay, so I was gonna ask you that - you had talked about that factness that you were dealing with robustness in biological systems, with the technical AI safety... Is that the crossover there, are we getting to that, or am I jumping in?
```

Comment:
Does he really say `that factness that you were dealing with robustness`? Doesn't seem like a phrase that Chris would say...

Reply:
@jerodsanto That is what I interpreted when I listened to the audio. But I will try again to correct it and put the right phrase.

Reply:
I checked it and that's the best guess I have as well. 🤔